fala/templates/macros/forms.html

```diff
@@ -54,9 +54,10 @@
       <div class="form-row {{ kwargs.row_class or '' }}">
         {% if field %}
           {% if field.errors and controlled_by and not controlled_by.errors or field.errors and not controlled_by %}
-            <input type="text" name="{{ field.name }}" value="{{ field.data or '' }}" class="govuk-input govuk-input--error govuk-!-width-one-third {{ field_attrs.class or '' }}" maxlength="{{ kwargs.max_length or '30' }}" id="id_{{ field.name }}">
+            <input type="text" name="{{ field.name }}" value="{{ field.data or '' }}" class="govuk-input govuk-input--error govuk-!-width-one-third {{ field_attrs.class or '' }}" maxlength="{{ kwargs.max_length or '30' }}" id="id_{{ field.name }}" aria-describedby="field-description-postcode">
```

Comment:
how come this branch of the condition doesn't need the field name check ie. `{% if field.name == 'postcode' %}aria-describedby="field-description-postcode"{% endif %}`? Unless I am totally misunderstanding what this branch does!

Reply:
We only want to apply it to the postcode input field as that is the only one that has help text. None of the others do. Because it's part of a loop, it actually applied the aria to every input.

Reply:
Okay, maybe I just don't fully understand how this code is working! My thinking was you might also want the condition on line 57 ie `{% if field.name == 'postcode' %}aria-describedby="field-description-postcode"{% endif %}`

I thought this was the field with an error summary attached or something, and you might still want the aria description attached to it but I could be totally wrong!

Reply:
Ohhhh no you're right. I've miss read this. I've added the if condition to the error summary to ensure it only adds the aria label if error is used.